### DISCUSSION TITLE
There is no diasurgical\devilution x set of folders

### DISCUSSION THREAD

**Discussion by Visurient** [diasurgical/DevilutionX#3353] [Q&A] [answered]
Windows version. I extracted the devilutionx-windows-x86_64.zip (not sure if thats even the one I'm supposed to be using) into a folder in C:\Program Files x86\ and did the copy this and that from GOG's Diablo I folder and booted up the game and everything was fine and I wanted to try out some of those .ini settings so I copied the Diablo 1.ini file into my Devilution folder and tried inputting manually some of the new commands but nothing worked and I checked online and learned there's some specific .ini in %Appdata%\diasurgical\devilution and I tried multiple ways to find this folder but it doesn't seem to exist did I do something wrong? is there some specific way I have to extract and setup devilutionx-windows-x86_64.zip?

**Comment by AJenbo** [comment, marked as answer]
How fare did you get? Try entering `%Appdata%` in to the address line in your file explorer
